code/src/components/newThoughts.css

```diff
@@ -0,0 +1,38 @@
+
```

Comment:
You had mentioned that you were struggling with formatting for responsiveness.  I wonder if the .thought-container and .new-thought-box classes are competing?  Also, maybe try a combination of width with view width (vw) as the unit of measurement and max-width to control for desktop sizing.